.devcontainer/config/sailbot_workspace.code-workspace

```diff
@@ -247,6 +250,17 @@
 				},
 				"problemMatcher": "$gcc"
 			},
+			{
+				"label": "Build Package",
```

Comment:
Also I'm thinking that it might be better to have a dedicated task for building each package. This way you don't have to type the package name every time.

Reply:
Used `pickString` to interactively select package from a list